tagcompare/output.py

```diff
@@ -8,11 +8,19 @@
 import glob
 from distutils import dir_util
 
+import logger
+
+
 BASE_DIR = os.path.abspath(os.path.dirname(__file__))
 OUTPUT_DIR = os.path.join(BASE_DIR, 'output')
 DEFAULT_BUILD_NAME = "default"
 DEFAULT_BUILD_PATH = os.path.join(OUTPUT_DIR, DEFAULT_BUILD_NAME)
 
+COMPARE_NAME = "compare"
```

Comment:
I don't really think it makes sense to make this part configurable though.  The settings are meant to be changed whereas these are meant to be just constants